renovate.json

```
@@ -18,6 +18,10 @@
       "groupName": "babel",
       "matchPackagePatterns": "@babel/*"
     },
+    {
+      "groupName": "D3",
+      "matchPackagePatterns": ["d3-", "@type/d3-*"]
```

Aren't we missing the wildcard here? 🤔 
```suggestion
      "matchPackagePatterns": ["d3-*", "@types/d3-*"]
```